src/forms/components/PassInput/index.js

```diff
@@ -0,0 +1,25 @@
+import React, {Fragment} from 'react';
+import {Text, TextInput} from 'react-native';
+
+import {styles} from './style';
+
+const PassInput = ({
+  label,
+  input: {onChange, ...restInput},
+  meta: {submitFailed, error},
+}) => {
+  return (
+    <Fragment>
+      <TextInput
+        secureTextEntry={true}
+        style={styles.input}
+        onChangeText={onChange}
+        {...restInput}
+        placeholder={label}
+      />
+      {submitFailed && error && <Text>{error}</Text>}
+    </Fragment>
+  );
+};
+
+export default PassInput;
```

Comment:
EmailInput and PassImput are exactly the same, the only difference is the secureTextEntry prop 🤔